lms/static/js/fixtures/bookmarks/bookmarks.html

```diff
@@ -0,0 +1,10 @@
+<div class="courseware-bookmarks-button">
+    <button type="button" class="bookmarks-button is-inactive" aria-pressed="false">
```

Comment:
Can we rename this class `bookmarks-button` to `bookmarks-list-button` or `show-bookmarks-list-button`.